src/pretty-dom.js

```diff
@@ -0,0 +1,22 @@
+import prettyFormat from 'pretty-format'
+
+const {DOMElement, DOMCollection} = prettyFormat.plugins
+
+function prettyDOM(htmlElement) {
+  const debugContent = prettyFormat(htmlElement, {
+    plugins: [DOMElement, DOMCollection],
+    printFunctionName: false,
+    highlight: true,
+  })
+  const maxLength = process.env.DEBUG_PRINT_LIMIT || 7000
```

Comment:
Perhaps we should accept this as an argument? I'm thinking that folks who are logging things themselves might want to control this more easily and in general probably want to see the whole thing. So perhaps we should only have this default for where it was already in use and the real default should be Infinity?

Reply:
Yes I agree with @kentcdodds. Currently when a test case is failing, its chipping of the container and I need to increase the `DEBUG_PRINT_LIMIT` to see the whole dom, which is causing me to re-run the case again.

Reply:
I changed it, and now the env var is only used internally when pretty printing out DOM inside thrown exceptions by the element getters. The exposed `prettyDOM` does not limit the output at all, and allows a second optional argument for the user to set a limit per call.